stdlib/private/SwiftReflectionTest/SwiftReflectionTest.swift

```diff
@@ -140,19 +119,95 @@ internal func getReflectionInfoForImage(atIndex i: UInt32) -> ReflectionInfo? {
                         reflstr: reflstr)
 }
 
-/// Get the TEXT segment location and size for a loaded image.
-///
-/// - Parameter i: The index of the loaded image as reported by Dyld.
-/// - Returns: The image name, address, and size.
-internal func getAddressInfoForImage(atIndex i: UInt32) ->
-  (name: String, address: UnsafeMutablePointer<UInt8>?, size: UInt) {
-  debugLog("BEGIN \(#function)"); defer { debugLog("END \(#function)") }
-  let header = unsafeBitCast(_dyld_get_image_header(i),
-    to: UnsafePointer<MachHeader>.self)
-  let name = String(validatingUTF8: _dyld_get_image_name(i)!)!
-  var size: UInt = 0
-  let address = getsegmentdata(header, "__TEXT", &size)
-  return (name, address, size)
+internal func getImageCount() -> UInt32 {
+  return _dyld_image_count()
+}
+
+let rtldDefault = UnsafeMutableRawPointer(bitPattern: Int(-2))
+#else // os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
+import SwiftShims
+import Glibc
```

Comment:
What about if we are on windows. @compnerd

Reply:
To be clear, I am fine if we just have an assert here or something like that so if someone tries to use it, we get an assertion or something. Just an else doesn't mean linux is the point.

Reply:
The Windows and Linux paths should be near identical.  The way that I had setup the reflection information on both of those platforms, each image has a structure with the addresses that is rebased by the loader.  At runtime, the information is registered with the runtime (see the `SwiftImageRegistrar`).  The section should have all the information that is needed.  I think that supporting Windows with the same path should be a very simple and straightforward thing to do and should be done as part of this change.

Reply:
@compnerd does Glibc exist on Windows? If it doesn't is there some analogous module that defines `stdin`, `stdout`, `errno,` etc I can import instead?

Reply:
There are examples of how to do portable file I/O in `stdlib/private/StdlibUnittest/StdlibUnittest.swift` and `tdlib/private/SwiftPrivateLibcExtras/Subprocess.swift`, and `stdlib/public/Platform/Platform.swift`. Is it possible to just import one of those?

Reply:
`errno` is largely unhelpful on Windows, the API matters significantly for the error cases.  `stdin`, `stdout`, and `stderr` are available in `MSVCRT`.